### ISSUE TITLE
dunst do not resize icon coming from mpdris2

### ISSUE THREAD

**Issue by nicoe** [dunst-project/dunst#859]
dunst do not resize the icon set by mpdris2 (I did not notice this behaviour before it might be more generic).

Here's the ouput of `dunst -print`
```
{
	appname: 'Music Player Daemon'
	summary: 'The Funny Bird'
	body: 'by Mercury Rev'
	icon: 'file:///media/Passport/music/Mercury Rev/1998 - Deserter's Songs/cover.jpg'
	raw_icon set: false
	icon_id: 'file:///media/Passport/music/Mercury Rev/1998 - Deserter's Songs/cover.jpg'
	desktop_entry: 'mpdris2'
	category:
	timeout: 10000
	urgency: LOW
	transient: 1
	formatted: 'The Funny Bird by Mercury Rev'
	fg: #000000
	bg: #f6fc7b
	frame: #aaaaaa
	fullscreen: show
	progress: -1
	stack_tag:
	id: 8
	actions: {}
	script_count: 0
}
```

### Installation info

- Version: `1.5.0 (2020-07-23)`
- Install type: `debian package`
- Distro and version: `debian package 1.5.0-1 `

**Comment by fwSmit** [comment]
What do you mean by not resizing? Can you send a screenshot?
Also, does it happen as well with `dunstify -i "file:///media/Passport/music/Mercury Rev/1998 - Deserter's Songs/cover.jpg" 'Notification'`.
Please send your dunstrc as well (upload it to gist.github.com).

**Comment by nicoe** [comment]
By not resizing I mean that the image is the size of the original image as you can see:

![screenshot](https://user-images.githubusercontent.com/44782/116875448-3e0bfe80-ac1b-11eb-8fe7-b6b40e751d60.png)

(the notification was generated by the dunstify command you gave).

Here's the dunstrc: https://gist.github.com/nicoe/6aa23ee75b88cd62ec8c633e7415eb6a

**Comment by fwSmit** [comment]
Max_size is not an config option. Use max_icon_size instead. Dunst currently doesn't warn you when you use non-existing options, but that's fixed in #803.

**Comment by nicoe** [comment]
* fwSmit  [2021-05-03 15:26 +0200]: 
>Max_size is not an config option. Use max_icon_size instead. Dunst currently
>doesn't warn you when you use non-existing options, but that's fixed in #803.

Oh thank you very much. Very sorry for the noise.
